configs/configgen.sh

```diff
@@ -3,14 +3,19 @@
 set -e
 
 SCRIPT_DIR=`dirname $0`
-BUILD_DIR=$2/configgen
+BUILD_DIR=/tmp/configgen
+OUT_DIR=$1
```

Comment:
It's probably fine in `/tmp` if that fails, we should be in the sandbox on multi-job/user systems anyway since your not disabling it for this rule.